src/pepr/operator/crd/sources/exemption/v1alpha1.ts

```diff
@@ -0,0 +1,99 @@
+import { V1CustomResourceDefinitionVersion, V1JSONSchemaProps } from "@kubernetes/client-node";
+
+export const v1alpha1: V1CustomResourceDefinitionVersion = {
+  name: "v1alpha1",
+  served: true,
+  storage: true,
+  additionalPrinterColumns: [
+    {
+      name: "Status",
+      type: "string",
+      description: "The status of the exemption",
+      jsonPath: ".status.phase",
+    },
+    {
+      name: "Age",
+      type: "date",
+      description: "The age of the exemption",
+      jsonPath: ".metadata.creationTimestamp",
+    },
+  ],
+  subresources: {
+    status: {},
+  },
+  schema: {
+    openAPIV3Schema: {
+      type: "object",
+      properties: {
+        status: {
+          type: "object",
+          properties: {
+            observedGeneration: {
+              type: "integer",
+            },
+            phase: {
+              enum: ["Pending", "Ready", "Failed"],
+              type: "string",
+            },
+          },
+        } as V1JSONSchemaProps,
+        spec: {
+          type: "object",
+          properties: {
+            exemptions: {
+              type: "array",
+              description: "Policy exemptions",
+              items: {
+                type: "object",
+                required: ["policies", "matcher"],
+                properties: {
+                  description: {
+                    type: "string",
+                    description:
+                      "A description of this exemption, this will become part of the exemption name",
+                  },
+                  policies: {
+                    description: "A list of policies to override",
+                    type: "array",
+                    items: {
+                      type: "string",
+                      enum: [
```

Comment:
Would be curious what others think, but I'd advocate we use `UPPER_SNAKE_CASE` for these values. 
```
                      enum: [
                        "DISALLOW_HOST_NAMESPACES",
                        "DISALLOW_NODEPORT_SERVICES",
                        "DISALLOW_PRIVILEGED",
                        "DISALLOW_SELINUX_OPTIONS",
                        "DROP_ALL_CAPABILITIES",
                        "REQUIRE_NON_ROOT_USER",
                        "RESTRICT_CAPABILITIES",
                        "RESTRICT_EXTERNAL_NAMES",
                        "RESTRICT_HOSTPATH_WRITE",
                        "RESTRICT_HOST_PORTS",
                        "RESTRICT_PROC_MOUNT",
                        "RESTRICT_SECCOMP",
                        "RESTRICT_SELINUX_TYPE",
                        "RESTRICT_VOLUME_TYPES",
                      ],
```

Reply:
I think constant case / `UPPER_SNAKE_CASE` does somewhat make sense based on general conventions I see upstream. Would point out that our codebase already has several enums that are `PascalCase` (`remoteGenerated` is a good example), so would be good to keep them standardized regardless of what we choose here - `PascalCase` seems to be somewhat common for enums too 🤷